cogboard-webapp/src/constants/index.js

```diff
@@ -331,5 +331,10 @@ export const NOTIFICATIONS = {
     type: 'error',
     message: message,
     duration: 3000
+  }),
+  CONFIG_LOADED: () => ({
+    type: 'info',
+    message: 'Config Loaded',
+    duration: 3000
```

Comment:
Cogboard will now display a message each time a new config is loaded in the background.